doc/install/dag-mcnp.rst

```diff
@@ -0,0 +1,107 @@
+..  |DAG-Code| replace:: DAG-MCNP5/6
+
+..  include:: header.txt
+
+..  include:: get_dagmc.txt
+
+..  _mcnp_patch:
+
+Apply the MCNP source code patch
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+DAGMC can be built with MCNP5_ or MCNP6_ or both, from Los Alamos National
+Laboratory. It is export-controlled software so you will need to request it from
+RSICC_.
+
+If you are building |DAG-Code|, you will need to copy the MCNP source code from
+the DVD into the DAGMC repository and apply a patch so it can be used with
+DAGMC. The patch you use must correspond to your version of MCNP. Currently
+supported versions of MCNP5 are 5.1.40, 5.1.51, 5.1.60
+::
+
+    $ cd mcnp/mcnp5
+    $ cp -r <path_to_dvd>/MCNP5/Source .
+    $ patch -p0 < patch/dagmc.5.1.60.patch
+
+Currently supported versions of MCNP6 are 6_beta2, 6.1, and 6.1.1beta.
+::
+
+    $ cd mcnp/mcnp6
+    $ cp -r <path_to_dvd>/MCNP6/Source .
+    $ patch -p0 < patch/dagmc.6.1.1beta.patch
+
+Assuming the patch or patches were succesfully applied, i.e. there were no
+warnings or errors, you are now ready to configure DAGMC to produce the desired
+build.
+
+..  include:: configure_dag-code_header.txt
+
+**Example 1:** Build the DAGMC interfaces and DAG-MCNP5, using the
+``$DATAPATH`` environment variable to specify the location of the MCNP data.
+::
+
+    $ cmake .. -DBUILD_MCNP5=ON \
+               -DCMAKE_INSTALL_PREFIX=$INSTALL_PATH
+
+**Example 2:** Build the DAGMC interfaces and DAG-MCNP5, assuming that the
+``$DATAPATH`` environment variable is undefined.
+::
+
+    $ cmake .. -DBUILD_MCNP5=ON \
+               -DMCNP5_DATAPATH=<path to MCNP data> \
+               -DCMAKE_INSTALL_PREFIX=$INSTALL_PATH
+
+**Example 3:** Build an MPI version of DAG-MCNP5.
+::
+
+    $ cmake .. -DBUILD_MCNP5=ON \
+               -DMPI_BUILD=ON \
+               -DCMAKE_INSTALL_PREFIX=$INSTALL_PATH
+
+**Example 4:** Build the DAGMC interfaces and DAG-MCNP6, using the
+``$DATAPATH`` environment variable to specify the location of the MCNP data.
+::
+
+    $ cmake .. -DBUILD_MCNP6=ON \
+               -DCMAKE_INSTALL_PREFIX=$INSTALL_PATH
+
+**Example 5:** Build the DAGMC interfaces and DAG-MCNP6, assuming that the
+``$DATAPATH`` environment variable is undefined.
+::
+
+    $ cmake .. -DBUILD_MCNP6=ON \
+               -DMCNP5_DATAPATH=<path to MCNP data> \
+               -DCMAKE_INSTALL_PREFIX=$INSTALL_PATH
+
+**Example 6:** Build an MPI version of DAG-MCNP6.
+::
+
+    $ cmake .. -DBUILD_MCNP6=ON \
+               -DMPI_BUILD=ON \
+               -DCMAKE_INSTALL_PREFIX=$INSTALL_PATH
+
+**Example 7:** Build both DAG-MCNP5 and DAG-MCNP6.
```

Comment:
No, always. I felt there were enough examples already so I didn't include any for building with MPI and using DMCNP5_DATAPATH or DMCNP6_DATAPATH. I feel this entire section might be a bit unnecessary; it might be better just to list the available cmake options.